<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBoxSpace">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="toolButtonRefresh">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="Experiment.qrc">
       <normaloff>:/openicon/icons/view-refresh-7.png</normaloff>:/openicon/icons/view-refresh-7.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Param</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="comboBoxParam">
     <property name="maxVisibleItems">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>From</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QDateTimeEdit" name="dateTimeEditFrom"/>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QComboBox" name="fromTimeCombo">
     <property name="maxVisibleItems">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="toTimeCheckBox">
     <property name="text">
      <string>To</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QDateTimeEdit" name="dateTimeEditTo"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>plot unit</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditPlotUnit"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Graph</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QComboBox" name="comboBoxPlotName"/>
   </item>
   <item row="7" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBoxSteps">
       <property name="text">
        <string>Plot as steps</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxUpdatePrevious">
       <property name="text">
        <string>Update previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxAutoUpdate">
       <property name="text">
        <string>auto update</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="8" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonUpdateAll">
       <property name="text">
        <string>Update All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCreatePlot">
       <property name="text">
        <string>Create Plot</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="9" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Experiment.qrc"/>
 </resources>
 <connections/>
</ui>
